src/content/2.4/Representable Functors.tex

```diff
@@ -245,7 +245,7 @@ \section{Representable Functors}\label{representable-functors}
 It must respect naturality conditions, and it must be the inverse of α:
 
 \begin{Verbatim}[commandchars=\\\{\}]
-α ◦ β = id = β ◦ α
+α  \ensuremath{\circ} β = id = β  \ensuremath{\circ} α
```

Comment:
What about moving also `\alpha`, `\beta`, `\gamma`, `\epsilon`, ... to their LaTex math counterparts?

Reply:
But the Unicode gives the same result (with the right packages like unicode-math, not sure it's setup here), and the source is more readable. Maybe we should move the Unicode alpha inside math mode.

Reply:
Maybe this should be part of https://github.com/hmemcpy/milewski-ctfp-pdf/pull/56

Reply:
Yeah, I'm inclined to leave the unicode symbols as is, if they render fine. Most of them do.